sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md

```diff
@@ -63,7 +63,7 @@ Use the [Azure CLI][azure_cli] snippet below to create/get client secret credent
 #### Create CertificateClient
 Once you've populated the **AZURE_CLIENT_ID**, **AZURE_CLIENT_SECRET** and **AZURE_TENANT_ID** environment variables and replaced **your-vault-url** with the above returned URI, you can create the [CertificateClient][certificate_client_class]:
 
-```C# CreateClient
+```C# Snippet:CreateClient
```

Comment:
Based on what you told me, these would need to be "scoped" to include "Certificate", "Key", etc., since they have to be unique across the repo.